Documentation/media-migration.md

```diff
@@ -1,151 +1,278 @@
+**AEPRoku SDK**
 
-In AEP Roku SDK, there are two Media trakcing APIs:
+```brightscript
+' Create SDK instance
+m.aepSdk = AdobeAEPSDKInit()
+```
 
-- `createMediaSession`
-- `sendMediaEven`
+> [!NOTE]
+> AEP SDK creates the taskNode internally.
 
-In the AEP Roku SDK, calling the above APIs is the recommended implementation path for sending Media events to Edge Network.
+## Start Media session
 
-## Initliaze SDK instances
+**Media SDK**
 
-| Media SDK      | AEP SDK |
-| ----------- | ----------- |
-| ' Create adbmobileTask node <br>  m.adbmobileTask = createObject("roSGNode","adbmobileTask")  <br> <br> ' Get AdobeMobile SG connector instance <br> m.adbmobile = ADBMobile().getADBMobileConnectorInstance(m.adbmobileTask) <br> | ' Retrieve the Adobe task node <br> m.adobeTaskNode = m.top.getScene().findNode("adobeTaskNode") <br> <br> ' Create a SDK instance <br> m.aepSdk = AdobeAEPSDKInit(m.adobeTaskNode) <br> |
-|    |         |
+``` brightscript
+' Use the helper method to create mediaInfo object
+mediaInfo = adb_media_init_mediainfo("test_media_name", "test_media_id", 10, ADBMobile().MEDIA_STREAM_TYPE_VOD, ADBMobile().MEDIA_TYPE_VIDEO)
 
-## Start the Media session
+' (Optional) Attach Standard metadata if any
+standardMetadata = {}
+standardMetadata[ADBMobile().MEDIA_VideoMetadataKeySHOW] = "sample show"
 
-- Media SDK
+mediaInfo[ADBMobile().MEDIA_STANDARD_MEDIA_METADATA] = standardMetadata
 
-``` brightscript
-mInfo = adb_media_init_mediainfo("test_media_name", "test_media_id", 10, "vod", ADBMobile().MEDIA_TYPE_VIDEO)
-mediaContextData = {}
-mediaContextData["videotype"] = "episode" 
-m.adbmobile.mediaTrackSessionStart(mInfo, mediaContextData)
+' (Optional) Create map for custom metadadata if any
+customMetadata = {}
+customMetadata["cmk1"] = "cmv1"
+
+' Call mediaTrackSessionStart API
+m.adbmobile.mediaTrackSessionStart(mediaInfo, customMetadata)
 ```
 
-- AEP SDK
+**AEPRoku SDK**
+
+> [!Note]
+> Checkout [sessionDetails](https://github.com/adobe/xdm/blob/master/docs/reference/datatypes/sessiondetails.schema.md) fieldgroup in the MediaAnalytics schema to know more about the required fields and StandardMetadata fields.
 
 ``` brightscript
-m.aepSdk.createMediaSession({
-    "xdm": {
-      "eventType": "media.sessionStart"
-      "mediaCollection": {
-        "playhead": 0,
-        "sessionDetails": {
-          "streamType": "video",
-          "friendlyName": "test_media_name",
-          "hasResume": false,
-          "name": "test_media_id",
-          "length": 10,
-          "contentType": "vod"
-        },
-        "customMetadata":{
-          "videotype":"episode" 
-        }
-      }
+sessionDetails = {
+    "streamType": "video",
+    "friendlyName": "test_media_name",
+    "name": "test_media_id",
+    "length": 10,
+    "contentType": "vod",
+
+    ''' (Optional) Attach Standard metadata if any
```

Comment:
`''' (Optional) Attach Standard metadata if any` -> `' (Optional) Attach Standard metadata if any`
https://developer.roku.com/docs/references/brightscript/language/expressions-variables-types.md#in-brightscript
Can we use only one apostrophe (') here?